scripts/buildImage.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -e
 
-cd "$(dirname -- "$0")/../"
```

Comment:
I expected to `cd` into the `scripts` directory, to be independent of the CWD location where this script is executed from.